tests/functional_small/ops/ops_eigen_vector.cc

```diff
@@ -284,73 +281,69 @@ TEST(ops_eigen, vector_update4)
 
 TEST(ops_eigen, vector_update_nan1)
 {
-  const auto nan = std::nan("0");
-  using V_t = Eigen::Matrix<double,3,1>;
-  V_t v; v << 1.,2.,3.;
-  V_t a; a << nan,nan,nan;
-  V_t b; b << nan,nan,nan;
-  V_t c; c << nan,nan,nan;
-  V_t d; d << nan,nan,nan;
-
-  pressio::ops::update(v, 1., a, 0.);
-  EXPECT_DOUBLE_EQ( v(0), 1.0);
-  EXPECT_DOUBLE_EQ( v(1), 2.0);
-  EXPECT_DOUBLE_EQ( v(2), 3.0);
-
-  pressio::ops::update(v, 1., a, 0., b, 0.);
-  EXPECT_DOUBLE_EQ( v(0), 1.0);
-  EXPECT_DOUBLE_EQ( v(1), 2.0);
-  EXPECT_DOUBLE_EQ( v(2), 3.0);
-
-  pressio::ops::update(v, 1., a, 0., b, 0., c, 0.);
-  EXPECT_DOUBLE_EQ( v(0), 1.0);
-  EXPECT_DOUBLE_EQ( v(1), 2.0);
-  EXPECT_DOUBLE_EQ( v(2), 3.0);
-
-  pressio::ops::update(v, 1., a, 0., b, 0., c, 0., d, 0.);
-  EXPECT_DOUBLE_EQ( v(0), 1.0);
-  EXPECT_DOUBLE_EQ( v(1), 2.0);
-  EXPECT_DOUBLE_EQ( v(2), 3.0);
+  V_t v, a, nan;
```

Comment:
can you please rename: `nan -> vecOfNans` so that it is clear this is a vector and there is no change to mistake this for a std::nan

Reply:
ok, fixed on 1767f32655f82ac4189933cd0a74418148c379bc